flixel/FlxSprite.hx

```diff
@@ -1207,6 +1225,7 @@ class FlxSprite extends FlxObject
 	
 	private function set_color(Color:FlxColor):Int
 	{
+		Color = 0x00FFFFFF & Color;
```

Comment:
I'm pretty sure the idea here is that the alpha part is not stripped away so this does not cause unexpected behavior:

```haxe
sprite.color = FlxColor.RED;
trace(sprite.color == FlxColor.RED); // you'd expect this to trace `true`
```

What edge cases are you talking about specifically?